.github/workflows/swift-toolchain.yml

```diff
@@ -3,6 +3,18 @@ name: Swift Toolchain Build
 on:
   workflow_call:
     inputs:
+      builder_os:
+        required: true
+        type: string
+
+      builder_arch:
+        required: true
+        type: string
```

Comment:
Please use standard terms, `builder` is not one of them: `build`, `host`, `target`.